device.py

```
@@ -0,0 +1,296 @@
+    def _normalize_token(self, token: str) -> str:
+        """Normalize JWT token by stripping optional Bearer prefix."""
+        if not token:
+            raise ValueError("JWT token is required")
+        token = token.strip()
+        if token.lower().startswith("bearer "):
+            return token.split(" ", 1)[1]
+        return token
+
+    def _generate_device_id(self) -> str:
+        """Generate unique device ID based on MAC address"""
+        import uuid
```

Ruff (0.14.14)

Redefinition of unused `uuid` from line 9: `uuid` redefined here

Remove definition: `uuid`

(F811)